runtime/src/snapshot_bank_utils.rs

```diff
@@ -522,36 +518,8 @@ fn verify_bank_against_expected_slot_hash(
 
     let bank_hash = bank.get_snapshot_hash();
     if bank_hash == snapshot_hash {
-        return Ok(());
-    }
-
-    // If the slots match but the hashes don't, there may be a mismatch between snapshot
-    // generation and snapshot load w.r.t. the snapshots_lt_hash cli args.
-
-    if bank
-        .feature_set
-        .is_active(&feature_set::snapshots_lt_hash::id())
```

Comment:
Does this mean we can't replay snapshot generated pre lt-hash feature is enabled with master now?